### PR TITLE
Add clear() method to the API

### PR DESCRIPTION
**Pull request by ivoanastacio**
I was also facing the need for a clear() method expressed in issue #33 and I ended up making a few small changes to add that functionality.

### CODE REVIEW

File: dist/angular-storage.js [hunk 1]
```diff
@@ -12,18 +12,25 @@ angular.module('angular-storage',
 
 angular.module('angular-storage.cookieStorage', [])
   .service('cookieStorage', ["$injector", function ($injector) {
-    var $cookieStore = $injector.get('$cookieStore');
+    var $cookies = $injector.get('$cookies');
```
Comment (AndreiRailean) [inline dist/angular-storage.js]: why are you renaming $cookieStore to $cookies ?
`$injector.get('$cookies')` is not failing?

https://docs.angularjs.org/api/ngCookies/service/$cookieStore

Comment (ivoanastacio) [inline dist/angular-storage.js]: The most recent Angular documentation states:

"Note: The $cookieStore service is deprecated. Please use the $cookies service instead."

**Comment by JackLeo** [comment]
are there anything else to do for this to be accepted or this plugin is considered dead given that the last interaction was Dec and PR was opened last year Aug...

**Comment by dentrinh** [comment]
I was about to make the same changes as this PR.  I've reviewed the PR and it looks good.  Can the conflicts be resolved and merged?

**Comment by ntotten** [comment]
Please rebase and we'll get it accepted.

**Comment by RodolfoSilva** [comment]
+1

**Pull request closed without merging**